include/ring-core/target.h

```diff
@@ -45,16 +58,11 @@
 #define OPENSSL_MIPS64
 #elif defined(__PPC64__) || defined(__powerpc64__)
 #define OPENSSL_64_BIT
-#elif (defined(__PPC__) || defined(__powerpc__)) && defined(_BIG_ENDIAN)
+#elif (defined(__PPC__) || defined(__powerpc__))
 #define OPENSSL_32_BIT
-#elif defined(__riscv) && __SIZEOF_POINTER__ == 8
-#define OPENSSL_64_BIT
-#define OPENSSL_RISCV64
 #elif defined(__s390x__)
 #define OPENSSL_64_BIT
 #define OPENSSL_S390X
-#elif defined(__wasm__)
-#define OPENSSL_32_BIT
 #else
 // Note BoringSSL only supports standard 32-bit and 64-bit two's-complement,
 // little-endian architectures. Functions will not produce the correct answer
```

Comment:
Please update comment to reflect that some big-endian targets are supported.

Reply:
I just removed the comment.